javascript/data.js

```diff
@@ -0,0 +1,52 @@
+const Data = {
+	speakers: [
+{
+	key: 0,
+	speakerName: 'Julie Sweet',
+	speakerPosition: 'Chief Executive Officer of Accenture',
+	speakerImage: './Assets/Images/Julia-Sweet.jpeg',
+	speakerBio: "American business executive. She is chief executive officer of Accenture, a multinational professional services company. She is also a member of the Global Management Committee for Accenture. According to The New York Times, she is 'one of the most powerful women in corporate America.'",
+},
+
+{
+	key: 1,
+	speakerName: 'David Essel',
+	speakerPosition: 'Master Business, Relationship, and Success Coach, Counselor',
+	speakerImage: './Assets/Images/david-essel.jpeg',
+	speakerBio: "David is number one best-selling author (10), counselor, master life coach,  international speaker and minister whose mission is to positively affect 2 million people or more every day, in every area of life, regardless of their current circumstances.",
+},
+
+{
+	key: 2,
+	speakerName: 'David Meinz',
+	speakerPosition: 'America Personal Health Improvement Expert',
+	speakerImage: './Assets/Images/David-Meinz.jpeg',
```

Comment:
- [ ] Good job dynamically rendering the speakers' section using JS. There are a few styling tweaks to be made here and you're good to go.
- You are missing the image decoration to the top left corner of the images. 

_See images below_

**Desktop Screens**
**Original** | **Your project**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/125090961-d0e67080-e084-11eb-8e33-9ef52a67bb52.png) | ![image](https://user-images.githubusercontent.com/49621279/125091200-1145ee80-e085-11eb-873b-128867e14308.png)

Reply:
I don't have that image unless you want me to use photoshop to add those decorations, which unfortunately I don't know how to use, if you have any inputs feel free to share.

Reply:
You can use the same decoration as the `main program` section here.

Reply:
> You can use the same decoration as the `main program` section here.

I could list ten folks from my cohort who submitted and got their project approved without using that design, but I won't go into that. I keep saying it, these changes are supposed to be optional,